### DISCUSSION TITLE
Need to get Customer Deposits against Sales Order

### DISCUSSION THREAD

**Discussion by zeeshan-ahmad-com** [netsuitephp/netsuite-php#108] [Q&A]
Hi,

I am working on a requirement to get All Customer Deposits made against Sale order by Soap API call. There is no class to search Customer deposits. I can only get a Customer Deposit by ID by RecordRef GET request.

Is there any way I can get all the Customer Deposits list for a specific Sale Oder. Please Help.

Thanks in Advance

**Comment by jrebs** [comment]
Looking at the schema browser, it seems like customer deposits are a Transaction type of record, so I would think you can use the sales order as a RecordRef and search for the appropriate type search for a customerdeposit record.
